indexer/feature_utils.cpp

```diff
@@ -464,6 +464,12 @@ string GetReadableWheelchairType(TypesHolder const & types)
   return readableTypes[0];
 }
 
+ftraits::WheelchairAvailability GetWheelchairType(TypesHolder const & types)
+{
+  auto const opt = ftraits::Wheelchair::GetValue(types);
+  return opt ? *opt : ftraits::WheelchairAvailability::No;
```

Comment:
Why missing availability is interpreted as No instead of Unknown?

Reply:
Added WheelchairAvailability::Unknown